cmd/template/cluster/provider/templates/aws/bastion_machine_deployment.yaml.tmpl

```diff
@@ -0,0 +1,38 @@
+apiVersion: cluster.x-k8s.io/v1alpha3
+kind: MachineDeployment
+metadata:
+  labels:
+    cluster.x-k8s.io/cluster-name: {{ .Name }}
+    cluster.x-k8s.io/role: bastion
+    cluster.x-k8s.io/watch-filter: capi
+    giantswarm.io/cluster: {{ .Name }}
+    giantswarm.io/organization: {{ .Owner }}
+    "release.giantswarm.io/version": "{{ .ReleaseVersion }}"
```

Comment:
why there is `""` around label name

Reply:
Copy-paste. Will clean it up to be consistent.

Reply:
Should be tidy now.